src/php/lib/Grpc/AbstractCall.php

```diff
@@ -121,6 +121,23 @@ public function cancel()
     }
 
     /**
+     * Serialize a message to the protobuf binary format
+     *
+     * @param mixed $data The Protobuf message
+     *
+     * @return string The protobuf binary format
+     */
+    protected function serializeMessage($data) {
+        // Proto3 implementation
+        if (method_exists($data, 'encode')) {
+            return $data->encode();
+        }
+
+        // Protobuf-PHP implementation
+        return $data->serialize();
```

Comment:
User needs the official `protobuf` and `protobuf-php`? So user need to install the two lib.